agent/agent-tooling/src/main/java/com/microsoft/applicationinsights/agent/internal/configuration/Configuration.java

```diff
@@ -257,6 +257,11 @@ public String toString() {
     // telemetry (they are much smaller so a larger queue size is ok)
     public int metricsExportQueueCapacity = 65536;
 
+    // unfortunately the Java SDK behavior has always been to report the "% Processor Time" number
+    // as "normalized" (divided by # of CPU cores), even though it should be non-normalized
+    // maybe this can be fixed in 4.0 (would be a breaking change)
+    public boolean backCompatNonNormalizedCpuPercentage;
```

Comment:
usedNormalizedCpuPercentage?

Reply:
ya, this is confusing, I added more comments, I'm not sure what to do about the name, we already report a normalized cpu percentage so I'm not sure `useNormalizedCpuPercentage` makes it any clearer.

maybe `useNormalizedValueForNonNormalizedCpuPercentage` (with default `true` representing the backwards compatible value)? kinda long but... 😂

Reply:
useNormalizedValueForNonNormalizedCpuPercentage sounds good although it's a bit long but it's clearer.